packages/components/pie-notification/src/index.ts

```diff
@@ -17,12 +28,14 @@ import '@justeattakeaway/pie-icons-webc/IconInfoCircle';
 import '@justeattakeaway/pie-icons-webc/IconAlertCircle';
 import '@justeattakeaway/pie-icons-webc/IconAlertTriangle';
 import '@justeattakeaway/pie-icons-webc/IconCheckCircle';
+import '@justeattakeaway/pie-button';
 
 // Valid values available to consumers
 export * from './defs';
 
-const componentSelector = 'pie-notification';
-const componentClass = 'c-notification';
+export interface NotificationEventDetail {
```

Comment:
do you think we can move it to "./defs"?

Reply:
Actually this should not exists anymore.